pkg/util/parallel/parallel.go

```diff
@@ -147,7 +147,7 @@ func DoTasks(ctx context.Context, numberOfTasks int, options DoTasksOptions, tas
 
 				for _, buf := range workersBuffs {
 					if buf != taskResult.buf {
-						processTaskResultData(ctx, []byte(buf.String()))
+						processTaskResultData(ctx, buf.Bytes())
```

Comment:
It is the breaking change. The `Bytes()` method returns the slice that will be changed after the next buffer modification.

https://golang.org/pkg/bytes/#Buffer.Bytes
```
Bytes returns a slice of length b.Len() holding the unread portion of the buffer. The slice is valid for use only until the next buffer modification (that is, only until the next call to a method like Read, Write, Reset, or Truncate). The slice aliases the buffer content at least until the next buffer modification, so immediate changes to the slice will affect the result of future reads.
```

Reply:
Yep, have missed that `buf` is still used later. So it's an incorrect change, thanks for spotting!